util/install.sh

```diff
@@ -174,15 +174,15 @@ function mn_deps {
                         python-pep8 ${PYPKG}-pexpect ${PYPKG}-tk
     else  # Debian/Ubuntu
         pf=pyflakes
-        # Starting around 20.04, installing pyflakes instead of pyflakes3
+        # Starting Ubuntu 18.04, installing pyflakes instead of pyflakes3
         # causes Python 2 to be installed, which is exactly NOT what we want.
-        if [ `expr $RELEASE '>=' 20.04` = "1" ]; then
-                pf=pyflakes3
+        if ( [[ $DIST == Ubuntu ]] && [[ `expr $RELEASE '>=' 18.04` == 1 ]] ) ||
+           ( [[ $DIST == Debian ]] && [[ $RELEASE -ge 10 ]] ); then
+            pf=pyflakes3
```

Comment:
> For python 2, the ubuntu 20 package is still called pep8, so this breaks the python 2 code check.
> It also appears that pep8 has been renamed pycodestyle for recent Ubuntu (and I presume debian) releases.

That's not the case.

* `pep8` package is an alias of `pycodestyle` that is for Python 3. It is available in Ubuntu 18, Ubuntu 20, Debian 11.
* `python-pep8` package is for Python 2. It is only available in Ubuntu 18.

The existing script always installs `pep8` regardless of Python version.
I'm changing it to `pycodestyle` to reduce indirection.